### ISSUE TITLE
with XState freeze

### ISSUE THREAD

**Issue by codev0** [lukehaas/RunJS#323]
I've tried to run first example from [xstate docs](https://xstate.js.org/docs/) then app frizzed

Here is the code:
```js
import { createMachine, interpret } from 'xstate';

// Stateless machine definition
// machine.transition(...) is a pure function used by the interpreter.
const toggleMachine = createMachine({
  id: 'toggle',
  initial: 'inactive',
  states: {
    inactive: { on: { TOGGLE: 'active' } },
    active: { on: { TOGGLE: 'inactive' } }
  }
});

// Machine instance with internal state
const toggleService = interpret(toggleMachine)
  .onTransition((state) => console.log(state.value))
  .start();
// => 'inactive'

toggleService.send('TOGGLE');
// => 'active'

toggleService.send('TOGGLE');
// => 'inactive'
```

**Comment by lukehaas** [comment]
@codev0 thanks for raising this, I'll look into it.

**Comment by lukehaas** [comment]
The issue here is that this code is outputting 342 MB of output text. I'll fix this by implementing a solution to truncate large output.

**Comment by pct196** [comment]
Hi, @lukehaas, do you have an update on this issue please?

**Comment by lukehaas** [comment]
@pct196 I have a fix ready to go out in the next release. This will likely be in either December or January.

**Comment by lukehaas** [comment]
@codev0 this is now fixed in version 2.2.2
